internal/k8s/informers.go

```diff
@@ -38,6 +39,7 @@ func DefaultResources() []schema.GroupVersionResource {
 		contour_api_v1alpha1.ExtensionServiceGVR,
 		corev1.SchemeGroupVersion.WithResource("services"),
 		v1beta1.SchemeGroupVersion.WithResource("ingresses"),
+		networking_v1.SchemeGroupVersion.WithResource("ingresses"),
```

Comment:
`v1` didn't exist until 1.19, so this will effectively bump our min supported Kube version to 1.19, won't it? https://github.com/projectcontour/contour/blob/main/cmd/contour/serve.go#L368 will `Fatal` if it doesn't exist.

Also, since all `Ingresses` will simultaneously be represented as `v1beta1` and `v1` Ingresses (assuming a 1.19-1.22 cluster that has both API versions), how do we avoid processing them twice?

Reply:
I thought about the version not being there working on IngressClasses, we can add the logic to check if the resource exists like we did for service-apis. I think we can use that to also check if v1 resources exist and do logic on that. I have new commit here in a second to show that.

Reply:
Yeah I think we'll need to add the existence check, will probably have to do the same eventually for `v1beta1` too since it'll be dropped in 1.22, if we want to support both 1.18 and 1.22 simultaneously.